spec/factories.rb

```diff
@@ -16,4 +16,19 @@
     state 'MA'
     zip_code '02113'
   end
+
+  factory :review do
+    sequence(:title) { |n| "#{n} This is my pizza slice" }
+    sequence(:body) do |n|
+      "#{n} Literally stumbled upon this place. I had been
+      hearing recommendations about Regina for months, and the one time I get
+      lost in Boston dumps me right on their doorstep. You can't ignore fate,
+      even when it's only 5pm!"
+    end
+    rating 4
+    after(:build) do |review|
+      review.pizzeria = FactoryGirl.create(:pizzeria)
+      review.user = FactoryGirl.create(:user)
+    end
```

Comment:
FactoryGirl has this functionality built-in so you don't have to use `after` callbacks:

```ruby
factory :review do
  sequence(:title) ...
  sequence(:body) ...
  rating 4
  # These two lines will create the associated objects automatically
  pizzeria
  user
end
```